### PR TITLE
The elements of the list can be checked and unchecked

### PR DESCRIPTION
**Pull request by ylruiz**
The following files were updated:
- constants/actionTypes.js 
- actions/todos.js
- reducers/todos.js
- components/MainSection.js
- App.js
- App.css (added styles for .todo  and .todo-completed)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -9,4 +9,18 @@
...
   color: white;
-}
\ No newline at end of file
+}
+
+/* 
+ * MainSection 
+ */
+
+.todo {
```
Comment (krvajal) [inline src/App.css]: Maybe move this to its own file?

File: src/components/MainSection.js [hunk 2]
```diff
@@ -1,15 +1,30 @@
 import React from 'react'
 
 const MainSection = (props) => {
+
+    const isCompleted = item => {
```
Comment (krvajal) [inline src/components/MainSection.js]: This can be one liner

File: src/components/MainSection.js [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
     return (
         <section>
             <ul>
                 {props.list.map(item => (
-                    <li key={item.id}>
-                        <label>
-                            {item.description}
-                        </label>
-                    </li>
+                    <label className={isCompleted(item)}>
```
Comment (krvajal) [inline src/components/MainSection.js]: Worth becoming its own component?

**Comment by ylruiz** [comment]
Can I merge this PR?

**Pull request merged**